lib/module.js

```diff
@@ -85,19 +85,11 @@ module.exports = async function gtmModule (_options) {
     console.warn('[@nuxtjs/gtm] head is provided as a function which is not supported by this module at the moment. Removing user-provided head.')
     this.options.head = {}
   }
-  this.options.head.script = this.options.head.script || []
-  if (options.nonce) {
-    this.options.head.script.push({
-      hid: options.scriptId,
-      nonce: options.nonce,
-      innerHTML: script
-    })
-  } else {
-    this.options.head.script.push({
-      hid: options.scriptId,
-      innerHTML: script
-    })
-  }
+  this.options.head.script.push({
```

Comment:
The `script` property might be missing in which case this code will crash. You need have a code that sets it to empty array first, similarly to what the old code did.

Reply:
oops, accident.